shorebird_code_push/pubspec.yaml

```diff
@@ -1,6 +1,7 @@
 name: shorebird_code_push
 description: Get info about your Shorebird code push app
```

Comment:
It's probably more "Control Shorebird code push within your app"?

Reply:
Done.